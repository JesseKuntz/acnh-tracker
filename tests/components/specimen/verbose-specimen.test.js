import { h } from 'preact';
import VerboseSpecimen from '../../../src/components/specimen/verbose-specimen';

import { shallow } from 'enzyme';

describe('VerboseSpecimen', () => {
  test('matches snapshot', () => {
    expect(
      shallow(
        <VerboseSpecimen
          type={'type'}
          image={'image'}
          name={'name'}
          location={'location'}
          time={'time'}
        />
      )
    ).toMatchSnapshot();
  });
});
