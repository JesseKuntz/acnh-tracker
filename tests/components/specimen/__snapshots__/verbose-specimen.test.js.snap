// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VerboseSpecimen matches snapshot 1`] = `
<div
  className="specimen-base"
>
  <picture>
    <source
      srcset="../../assets/type/webp/image.webp"
      type="image/webp"
    />
    <source
      srcset="../../assets/type/png/image.png"
      type="image/png"
    />
    <img
      className="logo"
      src="../../assets/type/png/image.png"
    />
  </picture>
  <div
    className="verbose-title"
  >
    name
  </div>
  <div
    className="info-container"
  >
    <div
      className="info-text"
    >
      <span
        className="bold"
      >
        Location:
      </span>
       
      location
    </div>
    <div>
      <span
        className="bold"
      >
        Time:
      </span>
       
      time
    </div>
  </div>
</div>
`;
